tests/improver-weighted-blending/12-model_blending.bats

```diff
@@ -31,14 +31,14 @@
 
 . $IMPROVER_DIR/tests/lib/utils
 
-@test "weighted-blending linear coordinate input input output" {
+@test "weighted-blending model blending" {
```

Comment:
The kgo for this doesn't seem to have a "Title" attribute which I thought should be added at some point?